packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/bucket-with-parameters.yaml

```diff
@@ -0,0 +1,33 @@
+Parameters:
+  BucketName: 
+    Default: MyS3Bucket
+    AllowedPattern: ^[a-zA-Z0-9]*$
+    ConstraintDescription: a string consisting only of alphanumeric characters
+    Description: The name of your bucket
+    MaxLength: "10"
```

Comment:
Why are these strings? They should be integers, right?

Reply:
cloudformation accepts both. This was a way to test that.

Reply:
OK. That's fine then :).